src/subscript/ri_wellmod/ri_wellmod.py

```diff
@@ -196,79 +243,29 @@ def launch_resinsight(console_mode: bool, command_line_parameters: List[str]):
             )
-            if (
-                wrapper
-            ):  # If already tried via wrapper, no need to search, just try again
-                logger.debug(
-                    "Trying to launch ResInsight wrapper again: %s", resinsight_exe
-                )
-                try:  # Second launch attempt via wrapper
-                    resinsight = rips.Instance.launch(
-                        resinsight_executable=resinsight_exe,
-                        console=console_mode,
-                        command_line_parameters=command_line_parameters,
-                    )
-                except Exception as any_exception:  # pylint: disable=broad-except
-                    Path(resinsight_exe).unlink()  # Delete wrapper
-                    logger.error(str(any_exception))
-                    return False
-            else:  # Not via wrapper - try to find a valid install and wrap it
-                resinsight_exe = find_and_wrap_resinsight_version(
-                    get_rips_version_triplet()
-                )
-                if not resinsight_exe:
-                    return False
-                wrapper = True
-                try:  # First launch attempt via wrapper
-                    resinsight = rips.Instance.launch(
-                        resinsight_executable=resinsight_exe,
-                        console=console_mode,
-                        command_line_parameters=command_line_parameters,
-                    )
-                except Exception as any_exception:  # pylint: disable=broad-except
-                    logger.error(str(any_exception))
-                    try:  # Second launch attempt via wrapper
-                        resinsight = rips.Instance.launch(
-                            resinsight_executable=resinsight_exe,
-                            console=console_mode,
-                            command_line_parameters=command_line_parameters,
-                        )
-                    except Exception as any_exception:  # pylint: disable=broad-except
-                        Path(resinsight_exe).unlink()
-                        logger.error(str(any_exception))
-                        return False
-        else:  # Not a version error, just try a second time
-            try:
-                resinsight = rips.Instance.launch(
-                    resinsight_executable=resinsight_exe,
-                    console=console_mode,
-                    command_line_parameters=command_line_parameters,
-                )
-            except Exception as any_exception:  # pylint: disable=broad-except
-                logger.error(str(any_exception))
-                return False
 
     if wrapper:
-        Path(resinsight_exe).unlink()
+        Path(resinsight_exe).unlink()  # Delete wrapper
 
     return resinsight
 
 
-def find_candidate_modules(top_path: Path) -> Set[str]:
+def find_candidate_modules(top_path: Path) -> Set:
```

Comment:
This looks like something that has gone wrong when rebasing.

Reply:
oh my.. seems like it.. sorry- willfix